src/pretix/control/templates/pretixcontrol/order/change_locale.html

```diff
@@ -0,0 +1,27 @@
+{% extends "pretixcontrol/event/base.html" %}
+{% load i18n %}
+{% load bootstrap3 %}
+{% block title %}
+    {% trans "Change locale information" %}
+{% endblock %}
+{% block content %}
+    <h1>
+        {% trans "Change locale information" %}
+    </h1>
+
```

Comment:
I think it would be nice to add a short explanation to the view, explaining the results of changing an order locale. I don't think that this is immediately obvious to a user.